cesanta/patch_sdk.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+xtensa-lx106-elf-objcopy --redefine-sym _read_r=_read_r_broken --redefine-sym _write_r=_write_r_broken --redefine-sym _lseek_r=_lseek_r_broken --redefine-sym _close_r=_close_r_broken ./../lib/libmain.a
```

Comment:
please add a comment above about what exactly is broken and at what version (i.e. something like " as of 1.2.0 espressif's implementations are stubs that just print the name of the function")